crates/sc-consensus-subspace-rpc/src/lib.rs

```diff
@@ -349,8 +349,21 @@ where
                                             return;
                                         }
                                     };
+                                let reward_address =
+                                    match FarmerPublicKey::from_slice(&solution.reward_address) {
+                                        Ok(public_key) => public_key,
+                                        Err(_) => {
+                                            warn!(
+                                                "Failed to convert reward address: {:?}",
+                                                solution.reward_address,
+                                            );
+                                            return;
+                                        }
```

Comment:
Let's print an error here as well, why is it ignored?

Reply:
The error type is actually an empty tuple